source/develop/release-management/features/network/provider-physical-network.md

```diff
@@ -109,3 +123,13 @@ Testing of this feature should cover:
 - VM connectivity to physical network, with both VLAN tagged and untagged networks.
 - Live migration of VMs attached to external networks.
 - In case OVS cluster was configured on 4.1 and only then upgraded to 4.2 (without any `setupNetworks` calls after the upgrade was done), previous tests should also pass.
+
+## External references
+
+[Provider Extended Attributes of Networks in OpenStack Networking API v2.0][1]
+
+[1]: https://developer.openstack.org/api-ref/network/v2/#provider-extended-attributes
+
+[Unrestricted Network Names][2]
+
+[2]: /develop/release-management/features/network/unrestricted-network-names/
```

Comment:
Link is invalid

Reply:
Would be valid if hosted, but I replace with a full URL.